Docs/Machine/FFNN.md

```diff
@@ -0,0 +1,91 @@
+# FFNN
+A feedforward neural network (FFNN) Machine. This machine is constructed by providing a sequence of layers from the ``layer`` class. Each layer implements a transformation such that the information is transformed sequentially as it moves from the input nodes through the hidden layers and to the output nodes.
```

Comment:
Didn't you have a few more comments about FFNNs in the current docs? :) We can copy them from the current website repo, in case...